.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
list-staged is configured in `package.json` to run prettier and eslint and reject the commit if any changes would be made.